dask-gateway/Dockerfile

```diff
@@ -2,27 +2,34 @@
 FROM debian:buster-slim as miniconda
 LABEL MAINTAINER="Jim Crist-Harif"
 
-ARG CONDA_VERSION=py38_4.8.3
-ARG CONDA_SHA256=879457af6a0bf5b34b48c12de31d4df0ee2f06a8e68768e5758c3293b2daf688
```

Comment:
Removal of checksums as amd64 / arm64 will have different checksums and it is a bit tedious to keep track of both.